examples/run_demo_ursula_fleet.py

```diff
@@ -17,15 +17,15 @@
 
 from twisted.internet import reactor
 
-from constant_sorrow.constants import TEMPORARY_DOMAIN
 from nucypher.characters.lawful import Ursula
+from nucypher.config.constants import APP_DIR
 
 FLEET_POPULATION = 12
 DEMO_NODE_STARTING_PORT = 11500
 
 ursula_maker = partial(Ursula, rest_host='127.0.0.1',
                        federated_only=True,
-                       domains=[TEMPORARY_DOMAIN]
+                       domains=[":TEMPORARY_DOMAIN:"]
```

Comment:
Why not use the constant?

Reply:
Yeah... I was getting an error when trying to do that when it casts the constant to bytes because it tries to call its `encode` method.  (Which, actually, might be something we want to consider adding in CS).  Change it back and see for yourself.  I haven't tracked down how it started.

Reply:
Maybe that's a reason for using an actual constant, and not a CS constant.